/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
/home/CEFCA/aaorsi/work/jpas_mock/src/load_oiidata.py:8: RuntimeWarning: divide by zero encountered in power
  return q0 * (Zgas/0.012)**g0
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-17:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9323355,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-13:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9282122,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-11:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9257584,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-16:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9332691,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-19:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9236018,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-12:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9235620,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-20:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9124971,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-10:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9221301,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-15:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9207820,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-18:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9217029,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-6:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8995899,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-7:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8977421,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-8:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8933528,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-14:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (9085788,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-5:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8918441,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8921495,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-4:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8946067,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-9:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8947713,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8883206,) (9475440,) 
./get_all_lines.py:122: RuntimeWarning: divide by zero encountered in log10
  log_dist = np.log10(realdist) + np.log10(mpc2cm_24)+24.0 + np.log10(1+GalArr['redshift'])
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "./get_all_lines.py", line 127, in read_lightcone_chunk
    log_Fline = LinesLumArr[:,il] + 40.0 - (np.log10(4*np.pi)+2*(log_dist))
ValueError: operands could not be broadcast together with shapes (8841919,) (9475440,) 
